packages/schematics/src/collection/lib/index.ts

```diff
@@ -232,49 +225,70 @@ function addChildren(schema: NormalizedSchema): Rule {
   };
 }
 
-export default function(schema: Schema): Rule {
-  return wrapIntoFormat(() => {
-    const options = normalizeOptions(schema);
-    const moduleFileName = `${toFileName(options.name)}.module`;
-    const modulePath = `${options.fullPath}/${moduleFileName}.ts`;
-    const indexFile = `libs/${toFileName(options.fullName)}/index.ts`;
+export interface LibSchema {
```

Comment:
individual schematics should not export tokens. if you think the functionality should be reused, moved into some shared file